docs/introduction/running-acceptance-tests.md

```diff
@@ -10,7 +10,7 @@ docker exec -it shopsys-framework-php-fpm bash
 
 !!! note
     For `selenium-server` to be able to connect to you `webserver` container and access your application, all domains should have URL set to `http://webserver:8000`.
-    This is done via parameter `%overwrite_domain_url%` defined in `config/parameters_test.yaml`.
+    This is done via parameter `%overwrite_domain_url%` defined in `.env.test` or `.env.test.local`.
```

Comment:
```suggestion
    This is done via parameter `OVERWRITE_DOMAIN_URL` defined in `.env.test` or `.env.test.local`.
```

Reply:
I don't see this applied, is it intentional or by mistake?

Reply:
Yah, my bad. Fixed 4dbd0653a093429922a5d1d0283aad87d3ba51ae